### DISCUSSION TITLE
American Charlie Hall invented what household item in 1968?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#321] [Galaxy Trivia] [answered]
**Hint:** 🌊🛏️

**Comment by thuongtruong1009** [comment]
Charlie Hall invented the "clapper" in 1968. The clapper is a sound-activated electrical switch that allows people to turn on or off their lights or other electrical devices by clapping their hands.

**Comment by teafoot** [comment, marked as answer]
waterbed

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/teafoot?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
